Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -349,18 +349,20 @@ public bool RemovePlayer()
         {
             if (LogFilter.Debug) Debug.Log("ClientScene.RemovePlayer() called with connection [" + Connection + "]");
 
-            if (Connection.Identity != null)
-            {
-                Connection.Send(new RemovePlayerMessage());
+            if (Connection == null)
+                return false;
```

Comment:
Should we throw exception instead?

Reply:
sounds good to me. added an exception and updated the test to check for it.